src/lib/commands.ts

```diff
@@ -31,14 +31,14 @@ export function getCategories(): Record<
       description: 'Custom Pages',
       commands: [],
     },
-    categories: {
-      description: 'Categories',
-      commands: [],
-    },
     versions: {
       description: 'Versions',
       commands: [],
     },
+    categories: {
```

Comment:
Rearranged this so the categories in the help screen align with the README.md command reference.